### ISSUE TITLE
Worth splitting VegAssist's accounts by language?

### ISSUE THREAD

**Issue by bradbeattie** [plorry/VegAssist#49]
In following the bot, I find a fair number of spanish tweets that I can't really help with. Would one bot per language be the right approach here, such that users might follow the languages they speak?

**Comment by hyperreality** [comment]
Yeah, good idea, I was thinking the same thing a while back, but the developers have been very quiet so I didn't bring it up.

I would be happy to run the Spanish/Italian version of the bot and also would be willing to continue development in general if the devs are currently too pushed for time.

**Comment by bradbeattie** [comment]
I'm working on it now. Will keep you posted. :)

**Comment by zhaey** [comment]
I've discussed this with plorry before, but not on github apparently. Splitting by language would probably work well for Spanish and other 'big' languages, but a Dutch bot, which would only tweet maybe once a week, would probably be to small to be worth hosting (or following).

**Comment by plorry** [comment]
Hey all, just seeing this now!
Yeah, we've discussed it before; I think either approach would work. Spanish for sure gets a lot of action. A few of the English followers have another language they're able to respond to. German & French don't really see much action.
The main reason for keeping it from one account was just to maintain a single follower base, so no bot goes lonely due to inactivity.
I figured the only downside to the multiple languages was, as @bradbeattie mentioned, that tweets will show up in your feed that you can't necessarily help with. If they became too numerous (or if enough people think they already _are_ too numerous), then I'd definitely be open to multiple bots.

Also, do feel free to deploy your own, enabling only the language file of your choice! I'd be happy to promote it with the English / main bot.

**Comment by moshpirit** [comment]
Since VegAssist doesn't RT anything but English tweets now, how can we improve such thing and what can we do? I am no developer but if I can help in anything... I do have an old Android phone I wouldn't mind using as a server (again, I have no idea the limitations for that, I am no dev.) BTW I'm Spanish in case this helps with Spanish related work
